src/components/Timeline.tsx

```diff
@@ -92,16 +92,17 @@ function renderTimelineEvents(route: Route | undefined, events: TimelineEvent[])
 interface TimelineProps {
   class?: string
   routeName: string
+  route: Accessor<Route | undefined>
```

Comment:
@incognitojam I couldn't figure out how to pass the non-accessor as it was always undefined, but maybe we want to pass the accessor by default? Have any preference?

Reply:
Edit: I could pass it, but the loading animation breaks